test.ts

```diff
@@ -129,6 +129,42 @@ Deno.test("main", async (t) => {
     await relay.shutdown();
 });
 
+// https://github.com/nostr-protocol/nips/blob/master/11.md
+Deno.test("NIP-11: Relay Information Document", async (t) => {
+    try {
+        await Deno.remove("test.sqlite");
```

Comment:
let's lift `test.sqlite` to a global const and change the first test to refer to this variable as well

Reply:
ok